go.work

```diff
@@ -1,15 +1,18 @@
 go 1.22.4
 
+// The `skip:golangci-lint` comment tag is used to exclude the package from the `golangci-lint` GitHub Action.
+// The module at the root of the repo (`.`) is excluded because ./pkg/... is included manually in the `golangci-lint` configuration.
+
 use (
-	.
+	. // skip:golangci-lint
 	./pkg/apimachinery
 	./pkg/apiserver
 	./pkg/build
-	./pkg/build/wire
+	./pkg/build/wire // skip:golangci-lint
 	./pkg/promlib
 	./pkg/semconv
 	./pkg/storage/unified/resource
-	./pkg/util/xorm
+	./pkg/util/xorm // skip:golangci-lint
```

Comment:
@ryantxu thoughts on this comment tag approach for excluding submodules in certain workflows?